src/pages/using-spark/guides/writing-meaningful-web-content-with-semantic-html.mdx

```diff
@@ -0,0 +1,250 @@
+---
+title: Writing Meaningful Web Content With Semantic HTML
+---
+
+import { SprkHeading } from '@sparkdesignsystem/spark-react';
+
+<span
+  id="writing-meaningful-web-content-with-semantic-html"
+  class="docs-b-Link--anchor"
+></span>
+<SprkHeading
+  element="h1"
+  additionalClasses="sprk-u-mbl sprk-u-Measure"
+  isPageTitle
+>
+  Writing Meaningful Web Content with Semantic HTML
+</SprkHeading>
+
+Typically, in a word document, writers bold or italicize text to add emphasis to the copy.
+
+When producing web content, **HTML** (Hyper Text Markup Language) **elements** define the structure, purpose and intended meaning of a page's web content.
```

Comment:
I think this would read better:
`When producing web content, [HTML elements](https://developer.mozilla.org/en-US/docs/Web/HTML) define the structure, purpose and intended meaning of a page's web content.`

With our intended audience, I think it's fine we don't spell out what HTML acronym is. It creates a smoother reading experience this way.